composer.json

```diff
@@ -12,16 +12,16 @@
         }
     },
     "require": {
-        "statamic/cms": "^5.38",
+        "statamic/cms": "dev-php84",
         "spatie/simple-excel": "^3.7",
         "symfony/dom-crawler": "^7.1",
         "pixelfear/composer-dist-plugin": "^0.1.5",
-        "laravel/framework": "11.*"
```

Comment:
In #21, whilst fixing some Testbench-related things, I accidentally added a requirement for Laravel 11. This PR reverts that.